app/lib/rails_i18n_manager/google_translate.rb

```diff
@@ -1,5 +1,6 @@
 module RailsI18nManager
   module GoogleTranslate
+    require 'easy_translate'
```

Comment:
According to general ruby standards, `require` statements should be at the top of the file not within a class or method. 

But I would prefer we do all of our dependency require statements in `lib/rails_i18n_manager/engine.rb`